bin/Phantom/submit_phantom.py

```diff
@@ -0,0 +1,468 @@
+# so there are two places
+# 
+# 
+# one for the traditional wmLHE workflow
+# 
+# and one for the new susy parameter scan workflow
+# 
+# for the wmLHE workflow: https://github.com/cms-sw/cmssw/blob/master/GeneratorInterface/LHEInterface/plugins/ExternalLHEProducer.cc#L253
+# 
+# https://github.com/cms-sw/cmssw/blob/master/GeneratorInterface/LHEInterface/plugins/ExternalLHEProducer.cc#L366
+# 
+# for the newer susy workflow which is only used for parameter scans so far:
+# https://github.com/cms-sw/cmssw/blob/master/GeneratorInterface/Core/interface/GeneratorFilter.h#L247
+# 
+# https://github.com/cms-sw/cmssw/blob/09c3fce6626f70fd04223e7dacebf0b485f73f54/GeneratorInterface/Core/src/BaseHadronizer.cc#L78
+
+
+import sys
+import os
+import commands
+from commands import getstatusoutput
+import datetime
+import argparse
+import datetime
+import math
+import ConfigParser
+import re
+import time
+
+
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+
+
+def replaceParameterInFile (inputFile, outputFile, substitute): 
+    f = open (inputFile)
+    lines = f.readlines ()
+    f.close ()
+    f = open (outputFile, 'w')
+    for line in lines:
+        if line.startswith ('*') or line.startswith (' ') or len (line) < 3 or line.startswith ('.') :
+            f.write (line)
+        else:
+            words = line.split (' ')
+            if words[0] in substitute.keys ():
+                f.write (words[0] + ' ' + substitute[words[0]])
+            else:
+                f.write (line) 
+    f.close ()
+
+
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+
+
+def execute (command, verbosity = False) :
+    if (verbosity == True):
+        print '---> running:'
+        print command
+    retCode = getstatusoutput (command)
+    if (verbosity == True):
+        for ri in retCode: print ri
+    return retCode
+
+
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+
+
+def findLhapdfUsed (makefilename):
+    f = open (makefilename, 'r')
+    for line in f.readlines ():
+        if line.startswith ('#') : continue
+        if re.match ('^\s*PDFLIBDIR\s*=', line):
+            return line.split()[2]
+    
+
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+
+
+def modifySubmitfileIntelCompiler (submitfilename, pdfgridfolder, phantompdflib):
+    submitfile = open (submitfilename, 'read')
+    lines = submitfile.readlines ()
+    submitfile.close ()
+    submitfile = open (submitfilename, 'write')
+    submitfile.write ('#!/bin/bash\n\n')
+    submitfile.write ('source /afs/cern.ch/sw/IntelSoftware/linux/setup.sh intel64\n')
+    submitfile.write ('source /afs/cern.ch/sw/IntelSoftware/linux/x86_64/xe2016/compilers_and_libraries/linux/bin/compilervars.csh intel64\n')
+    submitfile.write ('export LHAPDF=' + pdfgridfolder + '\n')
+    submitfile.write ('export PDFLIBDIR=/afs/cern.ch/work/b/ballest/public/phantom/LHAPDF-6.1.5_work/lib\n')
```

Comment:
Why private path?

Reply:
that's for old versions of phantom, which where compiled by the authors with an Intel compiler. The last version I am using (which is also the first one for the wmLHE production) is compiled with gFortran: if you check the script, you will see that the function for the intel compiler is not called in the main

Reply:
(I leave the old function for the smart user who might need to use an old release and would like to profit of the scripts anyhow)

Reply:
I don't refer to the Intel compiler but to the PDFLIBDIR.
Since AFS is not accessible in production this can create problems

Reply:
I understood, the two things are linked and this function is NOT used in the main, so it will not create any problems by construction. The function remains there for any users who wants to produce LHE files or calculate cross sections with the old versions, privately, using the same input cfg file as the new configuration.